packages/editor/src/editor-ui/plugin-toolbar/plugin-tool-menu/anchor-link-copy-tool.tsx

```diff
@@ -18,11 +17,7 @@ export function AnchorLinkCopyTool({
   const editorStrings = useEditorStrings()
   const { strings } = useInstanceData()
 
-  if (
-    !navigator.clipboard ||
-    !window.location.href.includes('add-revision') ||
-    !shouldUseFeature('editorAnchorLinkCopyTool')
-  ) {
+  if (!navigator.clipboard || !window.location.href.includes('add-revision')) {
```

Comment:
Maybe add a comment explaining why only on paths containing `add-revision`: 
Makes only sense when we have an entity id. We don't have one on `/___editor_preview` for example. Also in `serlo-editor-for-edusharing`.

Reply:
thanks!